pkg/storage/memory/memory.go

```diff
@@ -834,3 +857,7 @@ func (s *MemoryBackend) ListStores(ctx context.Context, options storage.ListStor
 func (s *MemoryBackend) IsReady(context.Context) (storage.ReadinessStatus, error) {
 	return storage.ReadinessStatus{IsReady: true}, nil
 }
+
+func (s *MemoryBackend) SerializeReadChangesContToken(next string, objType string) ([]byte, error) {
+	return []byte(fmt.Sprintf("%s|%s", next, objType)), nil
+}
```

Comment:
I see that the sql-based storages all use a standard continuation token struct and shared marshal/unmarshal functions, should we move those out of sqlcommon and into storage, then use it everywhere?